src/components/Tags/AddTags.tsx

```diff
@@ -24,7 +25,7 @@ export const AddTags = () => {
   const [selectedTags, setSelectedTags] = useState<Tag[]>([]);
 
   useEffect(() => {
-    const combinedTags: Tag[] = defaultTagsList.map((tag) => convertToTag(tag));
+    const combinedTags: Tag[] = DefaultTagsList.map((tag) => convertToTag(tag));
```

Comment:
can just pass the function when its just taking the callbacks args
```suggestion
    const combinedTags: Tag[] = DefaultTagsList.map(convertToTag);
```

Reply:
you could just pass thing to the useState
 
 const [combinedTagsList, setCombinedTagsList] = useState<Tag[]>(() => DefaultTagsList.map((tag) => convertToTag(tag)));

although doing that looks like setCombinedTagsList isn't needed either so you could just declare as a variable on its own outside of the use effect and also remove the useState for it